jarviscli/plugins/file_manager.py

```diff
@@ -0,0 +1,104 @@
+from plugin import plugin
+import os
+import shutil
+
+@plugin("file manage")
+class file_manage:
+    def __call__(self,jarvis,s):
+        self.get_file_directory(jarvis)
+        self.get_cmd(jarvis)
+
+        if self.cmd == "delete":
+            self.delete(jarvis,self.file)
+        elif self.cmd == "move":
+            self.move(jarvis,self.file)
+        elif self.cmd == "rename":
+            self.rename(jarvis,self.file)
+
+    def get_file_directory(self,jarvis):
+        self.file = jarvis.input("Enter the directory of the file you would like to edit: ")
+
+    def get_cmd(self,jarvis):
+        # function to find command to be performed to file
+
+        self.possibleCmds = ["delete","move","rename"]
+
+        cmdValid = False
+        while not cmdValid:
+            # iterate through possible commands and say each
+            jarvis.say("Commands Avaliable")
+
+            i = 1
+            for cmd in self.possibleCmds:
+                jarvis.say(str(i) + ". " + cmd)
+                i = i + 1
+
+            self.cmd = jarvis.input("Enter command to be performed: ")
+
+            # check if command is valid. If not, end cycle
+            if self.cmd not in self.possibleCmds:
+                jarvis.say("Invalid command")
+            else:
+                cmdValid = True
+
+    def delete(self,jarvis,file):
+        # function to delete files
+
+        # first, check if file exists
+        if os.path.exists(file):
+
+            yes = True
+            while yes:
+                # confirm that file should be deleted
+                confirmation = jarvis.input("Are you sure you want to delete this file? This cannot be undone. (y/n)").lower()
+
+                if confirmation == "y":
+                    # delete file
+                    os.remove(file)
```

Comment:
With os operation best try-except common Exception.

```
Some error occurred, please open an issue on github!
Here is error:

Traceback (most recent call last):
  File "jarviscli/CmdInterpreter.py", line 286, in try_do
    do(self, s)
  File "jarviscli/plugin.py", line 216, in run
    self.get_plugins(command).run(jarvis, new_s)
  File "jarviscli/plugin.py", line 208, in run
    self._backend[0](jarvis.get_api(), s)
  File "/home/phil/git/Jarvis/jarviscli/plugins/file_manager.py", line 12, in __call__
    self.delete(jarvis,self.file)
  File "/home/phil/git/Jarvis/jarviscli/plugins/file_manager.py", line 57, in delete
    os.remove(file)
IsADirectoryError: [Errno 21] Is a directory: '/tmp/test'
```

According to doc, os.remove may throw IsADirectoryError and  FileNotFoundError
https://docs.python.org/3.8/library/os.html#os.remove

(Simmilar move and ~~rename~~ feature) EDIT: For rename you already used try-except ;)